users/serializers_test.py

```diff
@@ -107,6 +107,20 @@ def test_valid_ca_postal_codes(sample_address, postal_code):
     assert LegalAddressSerializer(data=sample_address).is_valid()
 
 
+@pytest.mark.parametrize("postal_code", ["0M0 2K0", "J8R P7Q", "GAG 2B7", "L6L 122"])
```

Comment:
Can we add some more invalid parameters for testing.
e.g. `JBR-3P7`, `J8R P7`, `K2 2K9`

Reply:
For different lengths we have already test at [test for invalid codes ](https://github.com/mitodl/mitxpro/pull/787/files/41c43e726de9a133e77c9aec59fa2729e328dce6#diff-cc7ddfb36106bbe9117d834a925d6342R86).

I have added another for `K0M-2K0`